src/libgeotiff-1-libtoolfilemagic.patch

```diff
@@ -0,0 +1,12 @@
+diff -bur libgeotiff-1.4.0.orig/ltmain.sh libgeotiff-1.4.0/ltmain.sh
+--- libgeotiff-1.4.0.orig/ltmain.sh	2014-10-24 01:20:03.172985419 +0530
++++ libgeotiff-1.4.0/ltmain.sh	2014-10-24 01:58:50.983491081 +0530
+@@ -2560,7 +2560,7 @@
+     ;;
+   *ar\ archive*) # could be an import, or static
+     if eval $OBJDUMP -f $1 | $SED -e '10q' 2>/dev/null |
+-       $EGREP 'file format pe-i386(.*architecture: i386)?' >/dev/null ; then
++       $EGREP 'file format pe-i386(.*architecture: i386)?|pe-x86-64' >/dev/null ; then
```

Comment:
The best solution here is running `autoreconf -fi` before configuring.

Also, [libtool upstream](http://git.savannah.gnu.org/cgit/libtool.git/plain/build-aux/ltmain.in) changed it to

```diff
-       $EGREP 'file format pe-i386(.*architecture: i386)?' >/dev/null ; then
+       $EGREP 'file format (pei*-i386(.*architecture: i386)?|pe-arm-wince|pe-x86-64)' >/dev/null; then
```